server/services/statsService.js

```diff
@@ -84,5 +84,22 @@ module.exports = function createstatsService(statsClient) {
         total: onTime + notOnTime,
       }
     },
+
+    async getInitialCategorisationTprsTotals(startDate, endDate, prisonId, transactionalClient) {
```

Comment:
All the other methods have catType as a parameter. Was there a specific reason to have two separate methods here?

Reply:
Changed to have a single `getTprsTotals` fn that takes `catType`